.github/workflows/zizmor.yml

```diff
@@ -19,7 +19,7 @@ jobs:
         with:
           persist-credentials: false
       - name: Install the latest version of uv
-        uses: astral-sh/setup-uv@f0ec1fc3b38f5e7cd731bb6ce540c5af426746bb # v6.1.0
+        uses: astral-sh/setup-uv@445689ea25e0de0a23313031f5fe577c74ae45a1 # v6.3.0
```

Comment:
This is indeed the v6.3.0 tag: https://github.com/astral-sh/setup-uv/releases/tag/v6.3.0